lib/handlers/compile.js

```diff
@@ -363,6 +363,38 @@ export class CompileHandler {
         res.send(compiler.possibleArguments.getOptimizationArguments(this.getUsedOptions(req)));
     }
 
+    /**
+     * Search the available revisions for a particular compiler library.
+     * Supports pagination via offset and limit query params.
+     */
+    searchLibrary(req, res) {
+        const compiler = this.compilerFor(req);
+        if (!compiler || !compiler.isCompilerLibrary) {
+            return res.status(404).send({error: 'Compiler library not found!'});
+        }
+
+        let query = req.query.q || '';
```

Comment:
can both be `const` I think? thanks for all these great checks btw!

Reply:
Done